GDJS/GDJS/Events/CodeGeneration/ObjectCodeGenerator.cpp

```diff
@@ -127,20 +127,23 @@ CODE_NAMESPACE = CODE_NAMESPACE || {};
  * Object generated from OBJECT_FULL_NAME
  */
 CODE_NAMESPACE.RUNTIME_OBJECT_CLASSNAME = class RUNTIME_OBJECT_CLASSNAME extends gdjs.CustomRuntimeObject {
-  constructor(runtimeScene, objectData) {
-    super(runtimeScene, objectData);
-    this._runtimeScene = runtimeScene;
+  constructor(parentInstanceContainer, objectData) {
+    super(parentInstanceContainer, objectData);
+    this._parentInstanceContainer = parentInstanceContainer;
 
     this._onceTriggers = new gdjs.OnceTriggers();
-    this._behaviorData = {};
+    this._objectData = {};
```

Comment:
Isn't this overwriting `this._objectData` that would have been set by the `super` constructor?

Reply:
Indeed, the one from the super constructor is actually not useful, I removed it.